src/haxeLanguageServer/Context.hx

```diff
@@ -161,6 +163,16 @@ class Context {
         return resolve(null);
     }
 
+    function onExit(_) {
+        if (haxeServer != null) {
+            haxeServer.stop();
+            haxeServer = null;
+            process.exit(1);
```

Comment:
Why exit with  in this case?

Reply:
It's in the specification:

> Exit Notification (:arrow_right:)
> A notification to ask the server to exit its process. The server should exit with success code 0 if the shutdown request has been received before; otherwise with error code 1.

Reply:
Ah, I see.